README.md

```diff
@@ -56,7 +56,7 @@ APIs exposed by node-addon-api are generally used to create and
 manipulate JavaScript values. Concepts and operations generally map
 to ideas specified in the **ECMA262 Language Specification**.
 
-The [Node-API Resource](https://nodejs.github.io/node-addon-examples/) offers an
+The [Node-API Resource](https://nodejs.github.io/node-addon-examples/) offers an
```

Comment:
This change replaces a non-breaking space (ASCII 160) with a regular space (ASCII 32). LGTM.